jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -288,4 +298,8 @@ private Variant loadVariantWhenAuthorized(
         // unauthorized...
         return null;
     }
+
+    private String extractPushIdentifier(HttpServletRequest request) {
+        return request.getHeader("push-identifier");
```

Comment:
not sure, should we expose this on the registration metadata?

In the http header case, I think we would need a different name, to avoid
conflicts

On Tuesday, April 14, 2015, Erik Jan de Wit <notifications@github.com>
wrote:

> In
> jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523#discussion_r28312592>
> :
>
> > @@ -288,4 +298,8 @@ private Variant loadVariantWhenAuthorized(
> >          // unauthorized...
> >          return null;
> >      }
> > +
> > +    private String extractPushIdentifier(HttpServletRequest request) {
> > +        return request.getHeader("push-identifier");
>
> Do we want to put it into the header, or is this nicer in the json?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523/files#r28312592>
> .
>


-- 
Sent from Gmail Mobile